cfgov/unprocessed/js/modules/YoutubePlayer.js

```diff
@@ -60,6 +67,26 @@ var API = {
     }
   },
 
+  /**
+   * Load Youtube max res image if it exists.
+   * TODO: Replace this method by calling the Youtube data API.
+   * https://developers.google.com/youtube/v3/getting-started#fields
+   */
+  loadImage: function( ) {
+    if ( this.videoId ) {
+      var _image = this.childElements.image;
+      _image.classList.add( CLASSES.IMAGE_LOADING_STATE );
+      var image = document.createElement( 'img' )
+      image.onload = function onImageLoad() {
+        if ( image.naturalWidth && image.naturalWidth != 120 ) {
```

Comment:
It's crazy because the youtube image will 404 but not fire `onerror`. Will add a comment.